File: 06/todd.py
## OLD CODE

def questionSix():
        sumOfSquares = 0
        total = 0
        squareOfSum = 0
        for i in range(1, 101, 1):
            square = i * i
            total = total + square
        sumOfSquares = total
        suM = 0
        for i in range(1, 101, 1):
            suM = suM + i
        squareOfSum = suM * suM
        print(squareOfSum)
        if sumOfSquares > squareOfSum:
            difference = sumOfSquares = squareOfSum
        else:
            difference = squareOfSum - sumOfSquares
        print(difference)

File: 001-099/006/todd.c
#include <stdio.h>
#include <math.h> 

int main(void){
     int sumOfSquares, squareOfSum, i, difference;
     sumOfSquares = 0; squareOfSum = 0;
     for ( i = 1; i <= 100; i++ ){
         sumOfSquares+=(i * i);
         squareOfSum+=i;
     }
     printf("Question 6: %d\n", abs((squareOfSum * squareOfSum) - sumOfSquares));
}
